### DISCUSSION TITLE
Secret Login Token?

### DISCUSSION THREAD

**Discussion by Justin-Ruediger** [SchoolUtils/WebUntis#97] [Q&A]
Hey, I wonder if anybody could help me.
I am building an application where students can enter their webuntis data and share their timetables with friend, but for security reasons I don’t want to store their Untis password in my database. Is it possible to generate this secret token when they first create their account and from then on use this for requests, and if so how?

I am thankful for any answers or ideas.

**Comment by TheNoim** [comment]
What is the reason against a user simply scanning the QR code? The code contains all the necessary information, including the secret.

**Reply by wolflu05** [reply]
I Never got a qr, but isn't there also the password encoded init? I guess he meant saving the session, but this doesn't work as the session is only valid for 15min?

**Comment by Justin-Ruediger** [comment]
I wanted the user to be able to register without him needing to have a second device at hand to open and scan the QR code

**Comment by luwol03** [comment]
What do you mean by "Webentwickler data" (I'm german, but I dont know what that should be in terms of untis)?

**Reply by Justin-Ruediger** [reply]
Oh I guess that must have been a auto correction mistake I meant webUNTIS
